detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectXmlResolver.cs

```diff
@@ -104,7 +104,7 @@ public DependencyResult Process()
                                 
                                 if (!String.IsNullOrWhiteSpace(versionOverrideStr) && CheckVersionOverride)
                                 { 
-                                    addNugetDependency(tree, include.Value, versionStr);
+                                    addNugetDependency(tree, include.Value, versionOverrideStr);
```

Comment:
This was an incorrect variable thats why was getting NullException, but should be resolved now.